src/ticketandersencourse/service/TicketService.java

```diff
@@ -44,15 +53,42 @@ public static void main(String[] args) {
         setTicketList(tickets);
 
         Ticket empty = new Ticket();
-        System.out.println(empty);
+        //System.out.println(empty);
```

Comment:
Always remove all commented parts of code or add comment why they are commented before send PR for review